helm-charts/splunk-otel-collector/templates/clusterRoleBinding.yaml

```diff
@@ -3,6 +3,7 @@ kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ template "splunk-otel-collector.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
This is a cluster scoped object, no need to add namespace here.

Reply:
Removed